### ISSUE TITLE
q2vr played through Revive on WMR Samsung HMD: can't read GUI text 

### ISSUE THREAD

**Issue by BLZdvb** [dghost/quake2vr#40]
Love the project BTW. 
I'm playing with the default settings through Revive injection on a Samsung HMD (higher resolution than Oculus). So the GUI for the game menu is illegible (see attached). Which means I don't know how to increase GUI resolution (I haven't played the game in years). 
Noob question: is there a command that increases menu resolution that I can type into the console?
![20180225162450_1](https://user-images.githubusercontent.com/36825917/36646680-f64c9e86-1a48-11e8-8460-48a7cb50e25d.jpg)

**Comment by Raptoid** [comment]
A bit late but have you tried gl_hudscale? Otherwise I also found this. https://www.reddit.com/r/Vive/comments/6zwmpt/quake_2_tiny_menu_interface_problem/ Make sure to read the one about downloading the HD texture pack. I believe they're talking about the install mentioned on the modder's page:
http://www.malcolm-s.net/q2vr/
I downloaded the full shareware version, then copied my steam quake 2 files it says to use over that and it works, albeit with some crashes though I think that is due to me using revive.
